pkg/skaffold/runner/runner.go

```diff
@@ -139,15 +139,28 @@ func newTaggerForConfig(t v1alpha2.TagPolicy) (tag.Tagger, error) {
 	return nil, fmt.Errorf("Unknown tagger for strategy %s", t)
 }
 
-// Run runs the skaffold build and deploy pipeline.
-func (r *SkaffoldRunner) Run(ctx context.Context) error {
-	if r.opts.DevMode {
-		if r.opts.Cleanup {
-			return cleanUpOnCtrlC(ctx, r.dev, r.cleanup)
-		}
-		return r.dev(ctx)
+// Build builds the artifacts.
+func (r *SkaffoldRunner) Build(ctx context.Context) error {
```

Comment:
Since the builder is embedded in the runner, you can just call build straight from that. Then, our cmd/build would just look like

```
runner := NewForConfig()
runner.Build()
```

Or most likely you can reuse the private r.build here so that the UX looks similar.

Reply:
I'd rather try that in another PR. I've tried something similar and failed.